src/Apps/Artist/Routes/CV/CVItem.tsx

```diff
@@ -91,84 +91,89 @@ class CVItem extends Component<CVItemProps, CVItemState> {
-                  <Col>
-                    <Category size={xs ? "2" : "3"} weight="medium">
-                      {this.props.category}
-                    </Category>
-                    <Spacer mb={xs ? 0.5 : 1} />
-                  </Col>
-                </Row>
-              )}
-              {Object.keys(groupedByYear)
-                .sort()
-                .reverse()
-                .map((year, index) => {
-                  const isFirst = index === 0
-                  const yearGroup = groupedByYear[year]
-                  return xs ? (
-                    <Flex key={index}>
-                      <Year size="2" mr={1}>
-                        {year}
-                      </Year>
-                      <Box>{this.renderEntries(yearGroup, "2")}</Box>
-                    </Flex>
-                  ) : (
-                    <Row key={index}>
-                      {!sm &&
-                        !md && (
-                          <Col xl={2} lg={2}>
-                            {isFirst && (
-                              <Category size="3" weight="medium">
-                                {this.props.category}
-                              </Category>
-                            )}
-                          </Col>
+    return (
+      <CVItems className="cvItems">
+        <Media lessThan="lg">
+          <Row>
+            <Col>
+              <Category size={["2", "3"]} weight="medium">
+                {this.props.category}
+              </Category>
+              <Spacer mb={[0.5, 1]} />
+            </Col>
+          </Row>
+        </Media>
+        {Object.keys(groupedByYear)
+          .sort()
+          .reverse()
+          .map((year, index) => {
+            const isFirst = index === 0
+            const yearGroup = groupedByYear[year]
+            return (
+              <>
+                <Media at="xs">
+                  <Flex>
+                    <Year size="2" mr={1}>
+                      {year}
+                    </Year>
+                    <Box>{this.renderEntries(yearGroup, "2")}</Box>
+                  </Flex>
+                </Media>
+                <Media greaterThan="xs">
+                  <Row>
+                    <Media greaterThan="md">
+                      <Col xl={2} lg={2}>
+                        {isFirst && (
+                          <Category size="3" weight="medium">
+                            {this.props.category}
+                          </Category>
                         )}
-                      <Col lg={1} md={2} sm={2}>
-                        <Year mr={2} size="3">
-                          {year}
-                        </Year>
                       </Col>
-                      <Col md={8} lg={9} xl={9} sm={10}>
-                        {this.renderEntries(yearGroup)}
-                      </Col>
-                    </Row>
-                  )
-                })}
+                    </Media>
+                    <Col lg={1} md={2} sm={2}>
```

Comment:
This is backwards and goes from lowest to high -- by switching order it can be simplified to 
```
<Col sm={2} lg={1}>
```
since the sizes carry upwards until the next adjusted prop